doc/source/overview/high_level.md

```diff
@@ -1082,6 +1082,7 @@ that the ALE plots potentially miss details local to individual instances as the
 :align: center
 :alt: Ale plots for those features that the above counterfactuals have changed the most. 
 ```
+(similarity-explanations)=
 
 ### 5. Similarity explanations
```

Comment:
Technically not a change, but is the "TBA" in this section intended? Assuming the overview is yet to be written, but maybe we could just add a one-sentence description?

Reply:
I agree, will add a sentence.

Reply:
> Technically not a change, but is the "TBA" in this section intended? Assuming the overview is yet to be written, but maybe we could just add a one-sentence description?

Hey Ashley. I left it there as the section is only a placeholder for now. I guess we can add something like "Similarity explanations present instances in the training set that are similar to the instance of interest according to the model", but it's already in the table above.

Reply:
@ascillitoe added a paragraph and an image as we're lacking images in the docs.

Reply:
Looks great! One minor typo:
"which forms an explanations"